tests/ml_models/test_sentencetransformermodel_pytest.py

```diff
@@ -58,6 +58,8 @@ def test_check_attribute():
     assert test_model.model_id == "sentence-transformers/msmarco-distilbert-base-tas-b"
 
     default_folder = os.path.join(os.getcwd(), "model_files")
+    print(default_folder)
```

Comment:
is it better to remove the print message in this test file?

Reply:
Good catch, will remove.

And about the `three files added in the docs` question: these files were added by the sphinx due to converting the notebook to html. We didn't add any files on purpose.

Reply:
weird, previously the notebook read the zipfile of synthetic queries in the same way and it didn't add the unzipped .p files to the doc folder. The only concern is that I am not sure if we have permission to post the data file on the github. I believed I generated the file using the dataset package data.

Reply:
This is a binary file. I already uploaded a same synthetic query zip file for the integration test. I don't think that should be an issue. 

Also I'm planning to raise another PR to support other rest API. Can I remove the print message in that time?

Reply:
cool